src/raftstore/store/fsm/store.rs

```diff
@@ -467,6 +468,17 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             warn!("set thread priority for raftstore failed, error: {:?}", e);
         }
 
+        {
+            // entry_gc thread
+            let (tx, rx) = mpsc::channel();
```

Comment:
why not using Worker here?

Reply:
By now, this thread is too simple. We don't need to interact with it.

Reply:
Did you mean tikv::util::Worker? I think mpsc::channel is enough and also we can keep code simple. It's a little over-complex to use Worker and implement a Runnable for it.

Reply:
no, please keep the same style.

Reply:
@siddontang 
Worker use mutex and poll to handle task, which will bring in unnecessary overhead.
Using another thread to reclaim memory is a temporary solution until we can import better ones like memory pool.